servlet-feature-pack/galleon-feature-pack/src/main/resources/packages/product.conf/content/modules/system/layers/base/org/jboss/as/product/wildfly-web/module.xml

```diff
@@ -22,7 +22,7 @@
   ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
   -->
 
-<module xmlns="urn:jboss:module:1.5" name="org.jboss.as.product" slot="wildfly-web">
+<module xmlns="urn:jboss:module:1.9" name="org.jboss.as.product">
```

Comment:
Shouldn't the 'slot' value now be incorporated in the 'name'?  @ropalka ?

Reply:
@boris-unkel I looked and it should be made part of the name, separated by a colon:

name="org.jboss.as.product:wildfly-web"

That's not needed for cases where the slot attribute had the value 'main', but is needed for other values.

Reply:
Thanks, I improved that also. I'm wondering why microprofile health tests under java 8 seem to work now. Did I miss something?